lib/hades_web/router.ex

```diff
@@ -10,5 +10,6 @@ defmodule HadesWeb.Router do
     pipe_through :api
 
     get "/hello", HelloController, :hello
+    resources "/mentors", MentorController, only: [:index, :create, :show, :update]
   end
```

Comment:
Once the only action we should not use is `:delete`, I think we can use `except:`
i.e.:
```
resources "/mentors", MentorController, except: [:delete]
```

Reply:
Well, we also don't have `:new` and `:edit` (https://hexdocs.pm/phoenix/routing.html#resources)

In general I find `only:` more explicit than `except:`, it avoids any confusion regarding which routes are available.